Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -469,6 +469,12 @@ def _define_symbols():
 _define_symbols()
 
 
+def make_pen(color, width):
```

Comment:
I suppose we should better name it `_make_pen`, so we don't commit to keeping this function. Otherwise some add-on could rely on it.